### PR TITLE
Update deployment-overview.md

### PR DESCRIPTION
**Pull request by belasobral93**
Updating Prefect to include latest version info of the integration

### CODE REVIEW

File: website/docs/docs/deploy/deployment-overview.md [hunk 1]
```diff
@@ -52,11 +52,33 @@ For more details on both of these methods, including example implementations, ch
 
 ## Prefect
 
-If your organization is using [Prefect](https://www.prefect.io), use the [DbtShellTask](https://docs.prefect.io/api/latest/tasks/dbt.html#dbtshelltask) to schedule, execute and monitor your dbt runs.
+If your organization is using [Prefect]([url](https://www.prefect.io/)), the way you will run your jobs depends on the version you are on, and whether you are orchestrating dbt Cloud or core jobs. A variety of options are described below.
```
Comment (mirnawong1) [inline website/docs/docs/deploy/deployment-overview.md]: sugg clarifying whether it's dbt version.

```suggestion
If your organization is using [Prefect](https://www.prefect.io/), the way you will run your jobs depends on the dbt version you're on, and whether you're orchestrating dbt Cloud or dbt Core jobs. 

Review a variety of options described below.
```

File: website/docs/docs/deploy/deployment-overview.md [hunk 2]
```diff
@@ -52,11 +52,33 @@ For more details on both of these methods, including example implementations, ch
...
 
-Alternatively, you can use the supported [ShellTask](https://docs.prefect.io/api/latest/tasks/shell.html#shelltask) to execute dbt commands through the shell.
+<Lightbox src="https://user-images.githubusercontent.com/12350579/186269036-01f1090f-1887-433d-8331-ee7c8b315dd5.png" title="Prefect DAG using a dbt Cloud job run flow"/>
```
Comment (mirnawong1) [inline website/docs/docs/deploy/deployment-overview.md]: updating image url

```suggestion
<Lightbox src="/img/docs/running-a-dbt-project/prefect_dag_dbt_cloud.jpg" title="Prefect DAG using a dbt Cloud job run flow"/> 
```

File: website/docs/docs/deploy/deployment-overview.md [hunk 3]
```diff
@@ -52,11 +52,33 @@ For more details on both of these methods, including example implementations, ch
...
-Alternatively, you can use the supported [ShellTask](https://docs.prefect.io/api/latest/tasks/shell.html#shelltask) to execute dbt commands through the shell.
+<Lightbox src="https://user-images.githubusercontent.com/12350579/186269036-01f1090f-1887-433d-8331-ee7c8b315dd5.png" title="Prefect DAG using a dbt Cloud job run flow"/> 
+
+### On Prefect 2
+
+#### dbt Cloud
+
+Use the [trigger_dbt_cloud_job_run_and_wait_for_completion]([url](https://prefecthq.github.io/prefect-dbt/cloud/jobs/#prefect_dbt.cloud.jobs.trigger_dbt_cloud_job_run_and_wait_for_completion)) flow. As jobs are executing, you can poll dbt to see whether or not the job completes without failures, through the [prefect UI]([url](https://docs.prefect.io/ui/overview/)).
+
+<Lightbox src="https://user-images.githubusercontent.com/35778239/202576940-79adcc08-6412-4749-83ad-f0b66ba4cddc.png" title="dbt Cloud job triggered by Prefect"/>
```
Comment (mirnawong1) [inline website/docs/docs/deploy/deployment-overview.md]: updating image url

 ```suggestion
<Lightbox src="/img/docs/running-a-dbt-project/dbt_cloud_job_prefect.jpg" title="dbt Cloud job triggered by Prefect"/> 
```

File: website/docs/docs/deploy/deployment-overview.md [hunk 4]
```diff
@@ -52,11 +52,33 @@ For more details on both of these methods, including example implementations, ch
...
+If your organization is using [Prefect]([url](https://www.prefect.io/)), the way you will run your jobs depends on the version you are on, and whether you are orchestrating dbt Cloud or core jobs. A variety of options are described below.
 
-Alternatively, you can use the supported [ShellTask](https://docs.prefect.io/api/latest/tasks/shell.html#shelltask) to execute dbt commands through the shell.
+<Lightbox src="https://user-images.githubusercontent.com/12350579/186269036-01f1090f-1887-433d-8331-ee7c8b315dd5.png" title="Prefect DAG using a dbt Cloud job run flow"/> 
+
+### On Prefect 2
+
+#### dbt Cloud
+
+Use the [trigger_dbt_cloud_job_run_and_wait_for_completion]([url](https://prefecthq.github.io/prefect-dbt/cloud/jobs/#prefect_dbt.cloud.jobs.trigger_dbt_cloud_job_run_and_wait_for_completion)) flow. As jobs are executing, you can poll dbt to see whether or not the job completes without failures, through the [prefect UI]([url](https://docs.prefect.io/ui/overview/)).
```
Comment (mirnawong1) [inline website/docs/docs/deploy/deployment-overview.md]: hey @belasobral93 fixing the url as the deploy link was directing this to a 'page not found'. below is the URL guidance:

❌ [prefect UI]([url](https://docs.prefect.io/ui/overview/)).
✅ [prefect UI](https://docs.prefect.io/ui/overview/).

```suggestion
Use the [trigger_dbt_cloud_job_run_and_wait_for_completion](https://prefecthq.github.io/prefect-dbt/cloud/jobs/#prefect_dbt.cloud.jobs.trigger_dbt_cloud_job_run_and_wait_for_completion) flow. As jobs are executing, you can poll dbt to see whether or not the job completes without failures, through the [Prefect UI](https://docs.prefect.io/ui/overview/).
```

File: website/docs/docs/deploy/deployment-overview.md [hunk 5]
```diff
@@ -52,11 +52,33 @@ For more details on both of these methods, including example implementations, ch
...
+
+#### dbt Cloud
+
+Use the [trigger_dbt_cloud_job_run_and_wait_for_completion](https://prefecthq.github.io/prefect-dbt/cloud/jobs/#prefect_dbt.cloud.jobs.trigger_dbt_cloud_job_run_and_wait_for_completion) flow. As jobs are executing, you can poll dbt to see whether or not the job completes without failures, through the [Prefect UI](https://docs.prefect.io/ui/overview/).
+
+<Lightbox src="https://user-images.githubusercontent.com/35778239/202576940-79adcc08-6412-4749-83ad-f0b66ba4cddc.png" title="dbt Cloud job triggered by Prefect"/> 
+
+#### dbt Core
+
+Use the [trigger_dbt_cli_command]([url](https://prefecthq.github.io/prefect-dbt/cli/commands/#prefect_dbt.cli.commands.trigger_dbt_cli_command)) task.
```
Comment (mirnawong1) [inline website/docs/docs/deploy/deployment-overview.md]:
```suggestion
Use the [trigger_dbt_cli_command](https://prefecthq.github.io/prefect-dbt/cli/commands/#prefect_dbt.cli.commands.trigger_dbt_cli_command) task.
```

File: website/docs/docs/deploy/deployment-overview.md [hunk 6]
```diff
@@ -52,11 +52,33 @@ For more details on both of these methods, including example implementations, ch
...
+
+For details on both of these methods, see [prefect-dbt docs]([url](https://prefecthq.github.io/prefect-dbt/)).
```
Comment (mirnawong1) [inline website/docs/docs/deploy/deployment-overview.md]:
```suggestion
For details on both of these methods, see [prefect-dbt docs](https://prefecthq.github.io/prefect-dbt/).
```

File: website/docs/docs/deploy/deployment-overview.md [hunk 7]
```diff
@@ -52,11 +52,33 @@ For more details on both of these methods, including example implementations, ch
...
+
+### On Prefect 1
+
+#### dbt Cloud
+
+Trigger dbt Cloud jobs with the [DbtCloudRunJob]([url](https://docs.prefect.io/api/latest/tasks/dbt.html#dbtcloudrunjob)) task. Running this task will generate a markdown artifact viewable in the Prefect UI. The artifact will contain links to the dbt artifacts generate as a result of the job run.
```
Comment (mirnawong1) [inline website/docs/docs/deploy/deployment-overview.md]: i think this is meant to be generated?

```suggestion
Trigger dbt Cloud jobs with the [DbtCloudRunJob](https://docs.prefect.io/api/latest/tasks/dbt.html#dbtcloudrunjob) task. Running this task will generate a markdown artifact viewable in the Prefect UI. The artifact will contain links to the dbt artifacts generated as a result of the job run.
```

File: website/docs/docs/deploy/deployment-overview.md [hunk 8]
```diff
@@ -52,11 +52,33 @@ For more details on both of these methods, including example implementations, ch
...
+If your organization is using [Prefect]([url](https://www.prefect.io/)), the way you will run your jobs depends on the version you are on, and whether you are orchestrating dbt Cloud or core jobs. A variety of options are described below.
 
-Alternatively, you can use the supported [ShellTask](https://docs.prefect.io/api/latest/tasks/shell.html#shelltask) to execute dbt commands through the shell.
+<Lightbox src="https://user-images.githubusercontent.com/12350579/186269036-01f1090f-1887-433d-8331-ee7c8b315dd5.png" title="Prefect DAG using a dbt Cloud job run flow"/> 
+
+### On Prefect 2
+
+#### dbt Cloud
+
+Use the [trigger_dbt_cloud_job_run_and_wait_for_completion](https://prefecthq.github.io/prefect-dbt/cloud/jobs/#prefect_dbt.cloud.jobs.trigger_dbt_cloud_job_run_and_wait_for_completion) flow. As jobs are executing, you can poll dbt to see whether or not the job completes without failures, through the [Prefect UI](https://docs.prefect.io/ui/overview/).
```
Comment (mirnawong1) [inline website/docs/docs/deploy/deployment-overview.md]:
```suggestion
Use the [trigger_dbt_cloud_job_run_and_wait_for_completion](https://prefecthq.github.io/prefect-dbt/cloud/jobs/#prefect_dbt.cloud.jobs.trigger_dbt_cloud_job_run_and_wait_for_completion) flow. As jobs are executing, you can poll dbt to see whether or not the job completes without failures, through the [Prefect user interface (UI)](https://docs.prefect.io/ui/overview/).

```

File: website/docs/docs/deploy/deployment-overview.md [hunk 9]
```diff
@@ -52,11 +52,33 @@ For more details on both of these methods, including example implementations, ch
...
+For details on both of these methods, see [prefect-dbt docs]([url](https://prefecthq.github.io/prefect-dbt/)).
+
+### On Prefect 1
+
+#### dbt Cloud
+
+Trigger dbt Cloud jobs with the [DbtCloudRunJob]([url](https://docs.prefect.io/api/latest/tasks/dbt.html#dbtcloudrunjob)) task. Running this task will generate a markdown artifact viewable in the Prefect UI. The artifact will contain links to the dbt artifacts generate as a result of the job run.
+
+#### dbt Core
+Use the [DbtShellTask]([url](https://docs.prefect.io/api/latest/tasks/dbt.html#dbtshelltask)) to schedule, execute and monitor your dbt runs. Use the supported [ShellTask]([url](https://docs.prefect.io/api/latest/tasks/shell.html#shelltask)) to execute dbt commands through the shell.
```
Comment (mirnawong1) [inline website/docs/docs/deploy/deployment-overview.md]:
```suggestion
Use the [DbtShellTask](https://docs.prefect.io/api/latest/tasks/dbt.html#dbtshelltask) to schedule, execute, and monitor your dbt runs. Use the supported [ShellTask](https://docs.prefect.io/api/latest/tasks/shell.html#shelltask) to execute dbt commands through the shell.
```

**Review by mirnawong1 — CHANGES_REQUESTED** [review]
hey @belasobral93 - awesome job with this PR! appreciate you opening this up and I've left some suggestions for you - have a look and let me know what you think 🤗

**Review by belasobral93 — COMMENTED** [review]
@mirnawong1 looks great! thanks

**Review by mirnawong1 — APPROVED** [review]

**Pull request merged**